easybuild/easyconfigs/f/FLANN/FLANN-1.9.1_fix-cmake.patch

```diff
@@ -0,0 +1,27 @@
+# Flann fails to build with newer versions of CMake, see:
```

Comment:
please remove this file

Reply:
Erm, I need the `FLANN-1.8.4_fix-cmake.patch` file to build. Do you mean to remove that line?

Reply:
Done. I changed that to the generic patch file as above. It took me a moment to see what you are after, sorry.